eng/scripts/automation/GenerateAndBuildLib.ps1

```diff
@@ -891,15 +891,19 @@ function GeneratePackage()
         $ciFilePath = "sdk/$service/ci.mgmt.yml"
     }
 
+    # get the sdk version
+    $projectFile = Join-Path $srcPath "$packageName.csproj"
+    $version=([xml](Get-Content $projectFile)).Project.PropertyGroup.Version
```

Comment:
If that property doesn't exists I think this will throw an exception. Something worth testing.

Reply:
hello @weshaggard I add the check if the version node exist or not.